### PR TITLE
Fixup documentation briefly for `sortShorthand`

### PR DESCRIPTION
**Pull request by kylorhall-atlassian**
### What is this change?

Updating documentation around the `sortShorthand` config as I was confused myself setting it up in a new repo.

### PR checklist

- [x] ~~Updated or added applicable tests~~ – n/a
- [x] Updated the documentation in `website/`

### CODE REVIEW

File: website/packages/docs/src/pages/pkg-babel-plugin-strip-runtime.mdx [hunk 1]
```diff
@@ -116,14 +116,3 @@ See [Shorthand properties](/shorthand) for more information.
 
 - Type: `boolean`
 - Default: `false`
-
-#### sortShorthand
```
Comment (kylorhall-atlassian) [inline website/packages/docs/src/pages/pkg-babel-plugin-strip-runtime.mdx:120]: This is just a duplicate (the section above is the same)

File: website/packages/docs/src/pages/pkg-webpack-loader.mdx [hunk 2]
```diff
@@ -307,3 +307,18 @@ See [Shorthand properties](/shorthand) for more information.
 
 - Type: `boolean`
 - Default: `false`
+
+This should be set on both the `@compiled/webpack-loader`:
+
+```js
+{
+  loader: '@compiled/webpack-loader',
+  options: { sortShorthand: true }
```
Comment (kylorhall-atlassian) [inline website/packages/docs/src/pages/pkg-webpack-loader.mdx:316]: Is this even right?  It's a bit unclear to me…

Comment (dddlr) [reply website/packages/docs/src/pages/pkg-webpack-loader.mdx:316]: docs changes look reasonable, let me double check in the repo

Comment (dddlr) [reply website/packages/docs/src/pages/pkg-webpack-loader.mdx:316]: oh this is incorrect, `sortShorthand` exists for `CompiledExtractPlugin` but not `@compiled/webpack-loader`

see current docs https://compiledcssinjs.com/docs/pkg-webpack-loader where `sortShorthand` is listed under the options for `CompiledExtractPlugin` but not listed under the options for `@compiled/webpack-loader`

**Review by dddlr — CHANGES_REQUESTED** [review]
apart from https://github.com/atlassian-labs/compiled/pull/1718#discussion_r1796211884, lgtm

File: website/packages/docs/src/pages/pkg-webpack-loader.mdx [hunk 3]
```diff
@@ -307,3 +307,18 @@ See [Shorthand properties](/shorthand) for more information.
 
 - Type: `boolean`
```
Comment (dddlr) [inline website/packages/docs/src/pages/pkg-webpack-loader.mdx:308]: ah this is a small nitpick, but can we ensure the `Type: boolean` `Default: false` stuff is still placed at the end of the `sortShorthand` subsection?

File: website/packages/docs/src/pages/pkg-webpack-loader.mdx (hunk 2 above)
Comment (kylorhall-atlassian) [reply website/packages/docs/src/pages/pkg-webpack-loader.mdx:316]: Oh, I got how this documentation works now 😅   I'll just keep the example so someone doesn't misunderstand it like I did.

**Review by dddlr — APPROVED** [review]
👍

**Review by JakeLane — APPROVED** [review]

**Pull request merged**